xml/rmt_install.xml

```diff
@@ -31,7 +33,232 @@
+       </para>
+      </entry>
+      <entry>
+       <para>
+        Size of default repositories in GB
+       </para>
+      </entry>
+      <entry>
+       <para>
+        Size of all repositories in GB
+       </para>
+      </entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>
+       <para>
+        SLES/12.4/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+       17.84
+       </para>
+      </entry>
+      <entry>
+       <para>
+        47.37
+       </para>
+      </entry>
+     </row>
+     
+    <row>
+      <entry>
+       <para>
+        PackageHub/12.4/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+        13.82
+       </para>
+      </entry>
+      <entry>
+       <para>
+        28.04
+       </para>
+      </entry>
+     </row>
+     
+     <row>
+      <entry>
+       <para>
+        sle-ha/12.4/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+        0.32
+       </para>
+      </entry>
+      <entry>
+       <para>
+        5.53
+       </para>
+      </entry>
+     </row>    
 
+      <row>
+      <entry>
+       <para>
+        sle-sdk/12.4/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+        9.69
+       </para>
+      </entry>
+      <entry>
+       <para>
+       23.32
+       </para>
+      </entry>
+     </row> 
+     
+        <row>
+      <entry>
+       <para>
+        sle-module-public-cloud/12/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+        2.32
+       </para>
+      </entry>
+      <entry>
+       <para>
+       11.63
+       </para>
+      </entry>
+     </row> 
+     
+         <row>
+      <entry>
+       <para>
+        sle-module-basesystem/15/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+       12.64
+       </para>
+      </entry>
+      <entry>
+       <para>
+       32.41
+       </para>
+      </entry>
+     </row>  
+     
+         <row>
+      <entry>
+       <para>
+       sle-module-desktop-applications/15/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+       8.15
+       </para>
+      </entry>
+      <entry>
+       <para>
+       18.46
+       </para>
+      </entry>
+     </row>  
+     
+          <row>
+      <entry>
+       <para>
+       PackageHub/15/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+       29.98
+       </para>
+      </entry>
+      <entry>
+       <para>
+       34.52
+       </para>
+      </entry>
+     </row>     
+     
+           <row>
+      <entry>
+       <para>
+       sle-module-server-applications/15/x86_64
+       </para>
+      </entry>
+            <entry>
+       <para>
+       2.74
+       </para>
+      </entry>
+      <entry>
+       <para>
+       4.79
+       </para>
+      </entry>
+     </row>   
+     
+            <row>
+      <entry>
+       <para>
+      sle-module-web-scripting/15/x86_64
```

Comment:
`sle-module-web-scripting` -- friendly names instead of identifiers? The name of the extension is `Web and Scripting Module 15 x86_64` -- that's what is displayed in SCC.

Same applies for all other products.